pkg/config/linters.go

```diff
@@ -9,3 +14,52 @@ type Linters struct {
 
 	Presets []string
 }
+
+func (l *Linters) Validate() error {
+	if err := l.validateAllDisableEnableOptions(); err != nil {
+		return err
+	}
+
+	if err := l.validateDisabledAndEnabledAtOneMoment(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (l *Linters) validateAllDisableEnableOptions() error {
+	if l.EnableAll && l.DisableAll {
+		return errors.New("--enable-all and --disable-all options must not be combined")
+	}
+
+	if l.DisableAll {
+		if len(l.Enable) == 0 && len(l.Presets) == 0 {
+			return errors.New("all linters were disabled, but no one linter was enabled: must enable at least one")
+		}
+
+		if len(l.Disable) != 0 {
+			return fmt.Errorf("can't combine options --disable-all and --disable %s", l.Disable[0])
+		}
+	}
+
+	if l.EnableAll && len(l.Enable) != 0 && !l.Fast {
+		return fmt.Errorf("can't combine options --enable-all and --enable %s", l.Enable[0])
+	}
+
+	return nil
+}
+
+func (l *Linters) validateDisabledAndEnabledAtOneMoment() error {
+	enabledLintersSet := map[string]bool{}
+	for _, name := range l.Enable {
+		enabledLintersSet[name] = true
+	}
+
+	for _, name := range l.Disable {
+		if enabledLintersSet[name] {
+			return fmt.Errorf("linter %q can't be disabled and enabled at one moment", name)
+		}
+	}
```

Comment:
Perhaps we should refactor this to use `slices.Contains`?
```suggestion
	for _, name := range l.Disable {
		if slices.Contains(l.Enable, name) {
			return fmt.Errorf("linter %q can't be disabled and enabled at one moment", name)
		}
	}
```

The lookup performance for enabled linters is negligible in this context.

Reply:
We don't need to use the package `slices` everywhere :smile: 
The current algo is readable, there is no issue with it, so I will keep the current algo.